Mathlib/Geometry/RingedSpace/LocallyRingedSpace/HasColimits.lean

```diff
@@ -216,12 +223,16 @@ theorem imageBasicOpen_image_open :
 
 @[instance]
 theorem coequalizer_π_stalk_isLocalHom (x : Y) :
-    IsLocalHom ((coequalizer.π f.toShHom g.toShHom : _).stalkMap x) := by
+    IsLocalHom ((coequalizer.π f.toShHom g.toShHom : _).stalkMap x).hom := by
   constructor
   rintro a ha
-  rcases TopCat.Presheaf.germ_exist _ _ a with ⟨U, hU, s, rfl⟩
-  rw [PresheafedSpace.stalkMap_germ_apply
+  rcases TopCat.Presheaf.germ_exist (C := CommRingCat) _ _ a with ⟨U, hU, s, rfl⟩
+  -- need `erw` to see through `ConcreteCategory.instFunLike`
+  erw [PresheafedSpace.stalkMap_germ_apply
     (coequalizer.π (C := SheafedSpace _) f.toShHom g.toShHom) U _ hU] at ha
```

Comment:
The `erw` + `show` should be solvable by using a lemma specialized to `CommRingCat.Hom.hom`, but I don't know if that is worth the duplication.

Reply:
This one is solvable by two explicit `rw`s in the right directions. I think that's better, do you agree?